internal/parser/parser/testdata/html.sample.en.html

```
@@ -0,0 +1,11 @@
+<html><body>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)